content/docs/start/data-and-model-access.md

```diff
@@ -71,7 +71,7 @@ $ dvc import https://github.com/iterative/dataset-registry \
 
 This is similar to `dvc get` + `dvc add`, but the resulting `.dvc` files
 includes metadata to track changes in the source repository. This allows you to
-bring in changes from the data source later, using `dvc update`.
+bring in changes from the data source later using `dvc update`.
```

Comment:
Why? "source later using" could use a pause.

Reply:
basically a comma splice. Could parenthesise instead...

```suggestion
bring in changes from the data source later (using `dvc update`).
```

but actually also sounds better without any pause.

Reply:
Sorry again caught up with way too much recently.

Reply:
So again, you may be right (I disagree) but this one change seems stylistic i.e. could prob work one way or the other. Let's try to avoid such changes?

The changes where you rephrase things for clarity are much more impactful I think.